### ISSUE TITLE
Failed to download .NET 8.0.301-global

### ISSUE THREAD

**Issue by Arteiimis** [dotnet/vscode-dotnet-runtime#1819]
### Describe the bug.

After installing the plugin and selecting New Project, the .net sdk will automatically start downloading, and then the following error will occur.

### Please attach log file(s) here if you have any.

![image](https://github.com/dotnet/vscode-dotnet-runtime/assets/94288976/e4cfd215-97be-465a-a965-c9a2d987332c)

```console
ms-dotnettools.csdevkit requested to download the .NET SDK.
Downloading .NET version(s) 8.0.301-global~x64 ...Cancelled Installation of .NET 8.0.301-global.
The requested version ϵͳ�Ҳ���ָ����ע������ֵ��
 is invalid.
Error
Failed to download .NET 8.0.301-global:
.NET Acquisition Failed: The requested version ϵͳ�Ҳ���ָ����ע������ֵ��
 is invalid.
```
![image](https://github.com/dotnet/vscode-dotnet-runtime/assets/94288976/4cec23d1-d0ba-42c0-a2cf-131f9e6a724c)

### Reproduction Steps

just install the plugin then select new project

### Expected Behavior

successful installation

### .NET Install Tool Version

8.0.301

### Your operating system? Please provide more detail, such as the version and or distro above.

Windows

### VS Code Version

1.89.1

**Comment by Arteiimis** [comment]
After testing, directly from the official website to download dotnet sdk can be used normally, but scoop installed sdk can not be used, temporarily uncertain whether the plug-in problem or scoop version of dotnet problem.

**Comment by nagilson** [comment]
This is likely because your display language is not EN, which is a bug that should be fixed in 2.0.6.
